dev-python/libsass/libsass-0.19.4.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="A straightforward binding of libsass for Python"
+HOMEPAGE="https://github.com/sass/libsass-python"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="dev-libs/libsass
+		dev-python/six[${PYTHON_USEDEP}]"
+
+# Remove sassc, in favour of pysassc, see: https://github.com/sass/libsass-python/issues/134
+# This avoids a file collision with dev-lang/sassc
+PATCHES=( "${FILESDIR}"/remove-sassc.patch )
```

Comment:
Patch names should always include `${PN}` and version when it was introduced. So that patch name would be `libsass-0.19.4_rename_sassc.patch`, especially since it's supposed to be fixed upstream in the next version bump.